_posts/2021-03-27-RIP-pullmoll.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "RIP pullmoll"
+comments: true
+---
+
+<center>
+	<img src="assets/img/pullmoll.png" alt="Picture of Pullmoll">
+</center>
+
+Today, we were informed that our long time contributor and maintainer Jürgen
+Buchmüller (pullmoll) has passed away.
+
+With boundless dedication, Jürgen was one of the most prolific contributors to
+the Void project and sponsored the project financially, too.
+
+He invested lots of his time in keeping gcc up to date, supplied tons of
+patches to port packages to new compilers and platforms and also took care of
+simple updates of packages.
+
+He was not only a dedicated and driven developer in the Void Linux project, he
+also was one of the friendliest and warmest person we had the pleasure to work
```

Comment:
people instead of person, suggested by ifreund

Reply:
applied